src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.module.css

```diff
@@ -166,22 +163,19 @@
   }
 
   .ctaTitleBlock {
-    margin-top: 40px;
     max-width: 100%;
   }
 
   .leftTopImageWrapper {
-    display: none;
+    margin-right: -40px;
+    margin-top: 30px;
+    position: relative;
+    right: 0;
```

Comment:
It looks like these two properties are being set to undo the effects of the same properties being set outside of the media query. The fact that these are undoing the other properties is not obvious from just reading the code inside the media query here, and I myself didn't even realize it until viewing the full set of CSS properties being applied to the element in the browser.

To make this a bit more maintainable, could we alter the `position` and `right` CSS properties being applied to `.leftTopImageWrapper` outside of the media query by putting them inside of a `--laptop-and-up` media query? This way, the effects of the relative positioning are more self-contained and we don't have to worry about undoing them in the smaller screen sizes.